source/includes/options-mongodrdl-commands.yaml

```diff
@@ -0,0 +1,265 @@
+     mongosqld --schemaMode custom --schemaSource <schema-db>
+
+---
+program: mongodrdl
+name: download
+directive: commandoption
+description: |
+  Downloads a schema specified by its name or
+  :ref:`ObjectId <document-bson-type-object-id>` and prints it to the
+  console.
+
+  ``download`` accepts the following options:
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 20 80
+
+     * - Name
+       - Description
+
+     * - ``--schemaSource``
+       - *Required.* Specifies the database where the schema information
+         is stored.
+
+     * - ``--name``
+       - The name of the schema. To learn more about naming schemas,
+         see the :commandoption:`name-schema <mongodrdl --name-schema>`
+         command.
+
+         *Required* if ``--schema`` is not specified.
+
+     * - ``--schema``
+       - The :ref:`ObjectId <document-bson-type-object-id>` of the
+         schema.
+
+         *Required* if ``--name`` is not specified.
+  
+  For example, the following command prints the ``movies`` schema to the
+  console:
+
+  .. code-block:: sh
+
+     mongodrdl download --schemaSource schemas --name movies
+  
+  To store the schema in a ``.drdl`` file, cat the output to the desired
+  ``.drdl`` file:
+
+  .. code-block:: sh
+
+     mongodrdl download --schemaSource schemas --name movies > ./movies.drdl
+
+---
+program: mongodrdl
+name: delete
+directive: commandoption
+description: |
+  Deletes a schema specified by its name or
+  :ref:`ObjectId <document-bson-type-object-id>`. 
+
+  ``delete`` accepts the following options:
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 20 80
+
+     * - Name
+       - Description
+
+     * - ``--schemaSource``
+       - *Required.* Specifies the database where the schema information
+         is stored.
+
+     * - ``--name``
+       - The name of the schema. To learn more about naming schemas,
+         see the :commandoption:`name-schema <mongodrdl --name-schema>`
+         command.
+
+         *Required* if ``--schema`` is not specified.
+
+     * - ``--schema``
+       - The :ref:`ObjectId <document-bson-type-object-id>` of the
+         schema.
+
+         *Required* if ``--name`` is not specified.
+
+  For example, the following command deletes the
+  ``5d7941dc6a26a3d0fc397284`` schema in the ``schemas`` database:
+
+  .. code-block:: sh
+
+     mongodrdl delete --schemaSource schemas --schema 5d7941dc6a26a3d0fc397284
+---
+program: mongodrdl
+name: name-schema
+directive: commandoption
+description: |
+  Creates a new name for a schema or updates a schema's existing name.
+
+  .. include:: /includes/fact-mongodrdl-schema-name.rst
+
+  ``name-schema`` requires the following options:
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 30 70
+
+     * - Name
+       - Description
+
+     * - ``--schemaSource``
+       - *Required.* Specifies the database where the schema information
+         is stored.
+
+     * - ``--name``
+       - *Required.* The name of the schema.
```

Comment:
This one is tricky, since it could be a new name for an unnamed schema or a new name for a schema which already has a name. Maybe use "The new name for the schema"?